lib/index.js

```diff
@@ -4,6 +4,11 @@
  */
 import KEYS from "./visitor-keys.js";
 
+/**
+ * @typedef {{ [type: string]: ReadonlyArray<string> }} KeysStrict
+ * @typedef {{ readonly [type: string]: ReadonlyArray<string> }} KeysStrictReadonly
```

Comment:
Thoughts about renaming `KeysStrict` -> `VisitorKeysWritable`, and `KeysStrictReadonly` -> `VisitorKeys`?

`VisitorKeys` would then be the same name as in [`@types/eslint-visitor-keys`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/eslint-visitor-keys/index.d.ts), which might be helpful for packages that are using `eslint-visitor-keys` with types, such as [`vue-eslint-parser`](https://github.com/vuejs/vue-eslint-parser/blob/f7a7a1ee2e4d14ff7062a053301779b5ec3415da/src/ast/traverse.ts#L6).